index.js

```diff
@@ -191,5 +191,7 @@ module.exports = function (options) {
 
   });
 
+  dup.errors = errors;
```

Comment:
Since this isn't officially part of the API, let's change this to `dup.__errors = errors` (and reflect the name change in the CLI code, too).